### PR TITLE
Can we get some faster phpunit execution?

### PR DESCRIPTION
**Pull request by MorrisJobke**
* downstream of https://github.com/owncloud/core/pull/27113

### CODE REVIEW

File: tests/lib/Repair/RepairCollationTest.php [hunk 1]
```diff
@@ -28,15 +33,15 @@ public function getAllNonUTF8BinTables($connection) {
...
+class RepairCollationTest extends TestCase {
 
 	/**
 	 * @var TestCollationRepair
 	 */
 	private $repair;
 
 	/**
-	 * @var \Doctrine\DBAL\Connection
+	 * @var Connection | IDBConnection
```
Comment (nickvergessen) [inline tests/lib/Repair/RepairCollationTest.php]: spaces are actually invalidating the type hint

File: tests/phpunit-autotest.xml [hunk 2]
```diff
@@ -2,6 +2,8 @@
 <phpunit bootstrap="bootstrap.php"
 		 strict="true"
 		 verbose="true"
+		 backupGlobals="false"
+		 backupStaticAttributes="false"
```
Comment (nickvergessen) [inline tests/phpunit-autotest.xml]: This will fuck up hard with all the caches in our static classes...

Comment (LukasReschke) [inline tests/phpunit-autotest.xml]: So should we close this?

Comment (MorrisJobke) [inline tests/phpunit-autotest.xml]: I would keep the namespace cleanups and remove the php unit adjustments.

**Comment by MorrisJobke** [comment]
@nickvergessen @LukasReschke Better now?

**Review by LukasReschke — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Pull request merged**